odpf/siren/siren.proto

```diff
@@ -0,0 +1,333 @@
+syntax = "proto3";
+package odpf.siren;
```

Comment:
Let's use versioned directory structure and packages. `odpf/siren/v1/siren.proto`
This comes from buf.build recommendations https://docs.buf.build/best-practices/module-development/#package-versions

Reply:
correction, lets just use versioned dir for now, package name can remain same.

Reply:
Some confusion among us during the discussion. Let's use the version in the package name as well.